srcpkgs/flowblade/template

```diff
@@ -1,17 +1,28 @@
 # Template file for 'flowblade'
 pkgname=flowblade
-version=2.8.0.3
+version=2.14.0.2
 revision=1
 build_wrksrc=flowblade-trunk
 build_style=python3-module
 make_install_args="--install-lib=${DESTDIR}/usr/share/flowblade"
 hostmakedepends="python3-setuptools"
 # https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/DEPENDENCIES.md
-depends="frei0r-plugins gmic mlt-python3 python3-Pillow python3-dbus gtk+3
- python3-gobject python3-numpy swh-plugins"
+depends="frei0r-plugins gmic mlt7 mlt7-python3 python3-cairo python3-Pillow python3-gobject python3-numpy gtk+3
+ swh-plugins gdk-pixbuf pango librsvg SDL_image swh-plugins python3-libusb1"
 short_desc="Non-linear video editor for Linux"
 maintainer="shizonic <realtiaz@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://jliljebl.github.io/flowblade/"
-distfiles="https://github.com/jliljebl/flowblade/archive/v${version}.tar.gz"
-checksum=3753363d15caa6f1901118132a87f92d3b5e843c325775dbd83b30ee4d2a9bbf
+distfiles="https://github.com/jliljebl/flowblade/archive/v${version}.tar.gz>${pkgname}-v${version}.tar.gz"
+checksum=9ed5766328c90aa9c2a1511f0bd2a15c2495a3a437e484aa7d74b2fb0cd04ae6
+make_check=no # no tests to run
+
+do_patch() {
+	# Desktop file should run /usr/bin/flowblade.sh
+	vsed -i installdata/io.github.jliljebl.Flowblade.desktop -e 's#env.*#/usr/bin/flowblade.sh %f#'
+}
+
+post_install() {
+	# Enforce X11, otherwise it crashes on wayland
+	vbin "${FILESDIR}/enforce-x11.sh" flowblade.sh
+}
```

Comment:
```suggestion
post_install() {
	vbin "${FILESDIR}/flowblade-wayland"
}
```

the desktop file already sets the env var, so the script should only be necessary for running from cmdline: https://github.com/jliljebl/flowblade/blob/v2.14.0.2/flowblade-trunk/installdata/io.github.jliljebl.Flowblade.desktop#L6